pyproject.toml

```
@@ -88,8 +88,9 @@ dependencies = [
     "python-dateutil >= 2.9.0.post0",  # transitive dependency (beeware/briefcase#1428)
     "requests >= 2.28, < 3.0",
     "rich >= 12.6, < 14.0",
+    "tenacity >= 8.0, < 9.0",
     "tomli >= 2.0, < 3.0; python_version <= '3.10'",
-    "tomli_w >= 1.0, < 2.0",
+    "tomli_w >= 1.0, < 2.0"
```

Comma on the end is intentional.
```suggestion
    "tomli_w >= 1.0, < 2.0",
```